src/components/TodoItem.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/no-unused-state */
+/* eslint-disable react/state-in-constructor */
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+import { FaTrash } from 'react-icons/fa';
+import styles from './TodoItem.module.scss';
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  }
+
+  componentWillUnmount() {
+    console.log('Cleaning up...');
+  }
```

Comment:
- [ ] You may remove console.log because it should be used only for testing/debugging purposes.